rollrus.go

```diff
@@ -26,20 +26,47 @@ var wellKnownErrorFields = []string{
 // May be used as a rollbar client itself
 type Hook struct {
 	roll.Client
-	triggers []log.Level
+	triggers      []log.Level
+	ignoredErrors map[error]bool
```

Comment:
`map[error]struct{}`  is meh because the use will be `if _, ok` instead of `if ignoredErrors[err]`. Yay / nay?